service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/DiaryPlanStatementMapper.java

```diff
@@ -38,7 +38,8 @@
     private static final Mustache PLAN_STATEMENT_TEMPLATE = TemplateUtils.loadTemplate("ehr_plan_statement_template.mustache");
     private static final String EMPTY_DATE = "nullFlavor=\"UNK\"";
     private static final String FULL_DATE = "value=\"%s\"";
-    public static final String REASON_CODE_TEXT_FORMAT = "Reason Code: %s";
+    private static final String COMMA_SPACE = ", ";
```

Comment:
Optional: how about `REASON_CODE_SEPARATOR` or sth, to show the purpose of this field?